unittests/builtinhttp/builtinhttp.cpp

```diff
@@ -426,6 +426,38 @@ private Q_SLOTS:
         }
     }
 
+    // Test that that if the setUseWSAddressing is set int the client  then the SOAP action
+    // is really placed to the SOAP envelope's header and not embedded to the  Content-Type HTTP header
+    void testUseWSAddressingCall()
+    {
+        HttpServerThread server(countryResponse(), HttpServerThread::Public);
+        server.setClientUseWSAddressing();
+
+        qDebug() << "server ready, proceeding" << server.endPoint();
+        KDSoapClientInterface client(server.endPoint(), countryMessageNamespace());
+        client.setSendHTTPActionParameter();
+
+        client.setSoapVersion(KDSoapClientInterface::SOAP1_1);
+        QString action = QStringLiteral("http://localhost/getEmployeeCountry");
+        KDSoapPendingCall call = client.asyncCall(QStringLiteral("getEmployeeCountry"), countryMessage(), action);
+        QVERIFY(!call.isFinished());
+        QTest::qWait(1000);
```

Comment:
This makes the test really slow. Better wait "just the right amount".
One easy solution is to just use call() instead of asyncCall().

Reply:
I agree, fixed.